WORKSPACE

```diff
@@ -7,6 +7,64 @@ workspace(name = "mchristen")
 # file.  When the symbol is loaded you can use the rule.
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+http_archive(
```

Comment:
let stackb_rules_proto pick most of these dependencies for us to ensure compatibility :shrug: